components/NavigationBar/examples/NavigationBarTypicalUseExample.swift

```diff
@@ -32,8 +31,7 @@ open class NavigationBarTypicalUseSwiftExample: NavigationBarTypicalUseExample {
     let mutator = MDCNavigationBarTextColorAccessibilityMutator()
     mutator.mutate(navBar!)
 
-    let colorScheme = MDCSemanticColorScheme()
-    MDCNavigationBarColorThemer.applySemanticColorScheme(colorScheme, to: navBar!)
+    MDCNavigationBarColorThemer.applySemanticColorScheme(colorScheme!, to: navBar!)
```

Comment:
Why does this one need `!`?

Reply:
The interface itself is in an obj-c file

Reply:
Ah got it.